micronota/bfillings/tests/data/RF00522.cm

```diff
@@ -0,0 +1,377 @@
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     30   0.00695  6.36418  5.89551  6.00942     55 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     31   0.28484  1.50116  4.55452  4.23554     56 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     32   0.00695  6.36418  5.89551  6.00942     57 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     33   0.00695  6.36418  5.89551  6.00942     58 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     34   0.00695  6.36418  5.89551  6.00942     59 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     35   0.00695  6.36418  5.89551  6.00942     60 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     36   5.86314  0.01031  6.17552  5.23348     61 C - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     37   4.67285  2.69829  5.18174  0.08587     62 U - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     38   0.00695  6.36418  5.89551  6.00942     63 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     39   0.85668  4.47318  0.65957  3.05875     64 G - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     40   2.31421  3.54583  0.15364  4.21760     65 G - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     41   0.84819  1.94970  1.20225  2.04813     66 a - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     42   0.25613  3.07997  1.78183  4.45078     67 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     43   0.59154  2.05011  1.68125  2.02749     68 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     44   0.52361  2.14503  2.12445  1.76575     69 A - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
+     45   1.18601  2.00934  2.13451  0.81603     70 u - - :
+          1.38629  1.38629  1.38629  1.38629
+          0.00270  5.91620        *  1.46634  0.26236  0.00000        *
+//
```

Comment:
some test files are shared. put specific ones into their own dir, and put
shared ones in current dir?

On Fri, Oct 16, 2015 at 9:59 AM, mortonjt <notifications@github.com> wrote:

> In micronota/bfillings/tests/data/RF00522.cm
> <https://github.com/biocore/micronota/pull/21#discussion_r42264586>:
>
> > +     41   0.84819  1.94970  1.20225  2.04813     66 a - - :
> > +          1.38629  1.38629  1.38629  1.38629
> > +          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
> > +     42   0.25613  3.07997  1.78183  4.45078     67 A - - :
> > +          1.38629  1.38629  1.38629  1.38629
> > +          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
> > +     43   0.59154  2.05011  1.68125  2.02749     68 A - - :
> > +          1.38629  1.38629  1.38629  1.38629
> > +          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
> > +     44   0.52361  2.14503  2.12445  1.76575     69 A - - :
> > +          1.38629  1.38629  1.38629  1.38629
> > +          0.00539  5.91889  5.91889  1.46634  0.26236  1.09861  0.40547
> > +     45   1.18601  2.00934  2.13451  0.81603     70 u - - :
> > +          1.38629  1.38629  1.38629  1.38629
> > +          0.00270  5.91620        *  1.46634  0.26236  0.00000        *
> > +//
>
> What do you think about the idea of storing each of these files into their
> own directory?
> It might get crazy once we have test files for each tool.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/21/files#r42264586>.
>